include/swift/SILOptimizer/Differentiation/DifferentiationInvoker.h

```diff
@@ -71,8 +71,17 @@ struct DifferentiationInvoker {
 
     /// The parent `apply` instruction and the witness associated with the
     /// `IndirectDifferentiation` case.
+#if defined(__FreeBSD__)
```

Comment:
Why not just do this always. Then we don't have conditional code.

Reply:
Happy to change it.